requirements/tests.txt

```diff
@@ -1,7 +1,8 @@
 coveralls==1.5.1
 mock==2.0.0
-mozfile >= 1.1
-mozprocess >= 0.22
+mozfile==1.1
+mozprocess==0.22
```

Comment:
I don't see this package used directly anymore. Is there an indirect import from one of the other packages?

Reply:
I could not find any use of the same.

Reply:
So I would say lets get it removed.